src/index.html

```diff
@@ -4,9 +4,301 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin="anonymous">
+    <link href="https://fonts.googleapis.com/css2?family=Cinzel:wght@400;700&family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a href="index.html" class="logo">
+        <img
+          src="images/logo.png"
+          alt="logo_Moyo"
+          class="Moyo_img"
+        >
+      </a>
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a class="nav__link is-active" href="index.html" >Apple</a>
+          </li>
+          <li class="nav__item">
+            <a class="nav__link" href="index.html">Samsung</a>
+          </li>
+          <li class="nav__item">
+            <a class="nav__link" href="index.html">Smartphones</a>
+          </li>
+          <li class="nav__item">
+            <a
+              class="nav__link"
+              href="index.html"
+              data-qa="nav-hover"
+            >
+              Laptops & Computers
+            </a>
+          </li>
+          <li class="nav__item">
+            <a class="nav__link" href="index.html">Gadgets</a>
+          </li>
+          <li class="nav__item">
+            <a class="nav__link" href="index.html">Tablets</a>
+          </li>
+          <li class="nav__item">
+            <a class="nav__link" href="index.html">Photo</a>
+          </li>
+          <li class="nav__item">
+            <a class="nav__link" href="index.html">Video</a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+    <main>
+      <div class="catalog">
+        <div class="card" data-qa="card">
```

Comment:
```suggestion
        <article class="card" data-qa="card">
```
Consider leveraging semantic tags